### ISSUE TITLE
Remove Stored Project - inactive on MyProject QfieldCloud projects

### ISSUE THREAD

**Issue by jpdupuy** [opengisch/QField#5600]
### Describe the issue
Long press on a project item in the QFieldCloud Projects / MyProject  does not work (no pop-up with usual options open proect/Remove Stored project) on Galaxy Tab PRO although the project has been already downloaded. 
The selected project does not appear in the recent project list (seems limited to 5 items)

### Reproduction steps
1. [Go to 'parameters / home /QFieldCloud Projects / MyProjects']
2. [Long press on a project items that has been already downloaded]

### Expected behavior
A pop-up should appear with 2 options (open project / remove stored project)

### Observed behavior
pop-up does not show up

### Screenshots and GIFs
[If applicable, add screenshots and GIFs to help explain your problem.]

### QGIS (please complete the following information)
 - OS: [Windows]
 - QGIS Version [3.34.4]
 - QFieldSync Version [e.g.4.10.1]

### QField (please complete the following information)
 - Device: [Galaxy Tab PRO]
 - OS: [e.g. Android 14]
 - QField version: [3.3.11 Darien]

### Additional information
* Problem started happening recently, didn't happen in an older version of QField: [Yes]
* Problem can be reliably reproduced, doesn't happen randomly: [Yes]
* Problem happens with all files and projects, not only some files or projects: [Yes]
* A list of 18 project files stored on QFieldCloud, 7 have already downloaded to the tablet. Only 5 appear in the recent projects.
* This problem does not appear on my Galaxy A55 5G smartphone

[If the problem happened with QFieldCloud, please add your username and project name.]

**Comment by nirvn** [comment]
@jpdupuy , long pressing with finger or stylus? Can't replicate on my end with QField 3.3.11.

**Comment by jpdupuy** [comment]
With the stylus. It does work on other projects never downloaded
________________________________
From: Mathieu Pellerin ***@***.***>
Date: ven. 30 août 2024, 15:40
To: opengisch/QField ***@***.***>
CC: DUPUY Jean-Pierre ***@***.***>, Mention ***@***.***>
Subject: Re: [opengisch/QField] Remove Stored Project - inactive on MyProject QfieldCloud projects (Issue #5600)


@jpdupuy<https://github.com/jpdupuy> , long pressing with finger or stylus? Can't replicate on my end with QField 3.3.11.

**Comment by nirvn** [comment]
@jpdupuy , can you confirm whether it works with your finger? If it doesn't, a screencast would be helpful. Thanks.

**Comment by jpdupuy** [comment]
@nirvn It does not work either with my finger.

The 2 options (open project / remove stored project) are inactive with a long press when the project has been already downloaded.
I noticed that, applying a short press, you open and display the project (normal behavior).
If you go back to "parameters / home /QFieldCloud Projects / MyProjects" then the long press will work (both with stylus and finger).

I was able to reproduce systematically this problem. First of all you need to logout. then login in. Refreshing the project list will not solve this issue. This way you will be able to fix this issue in the code.

**Comment by nirvn** [comment]
@jpdupuy , OK, we'll need a screencast I'm afraid, I can't replicate locally.

**Comment by jpdupuy** [comment]
@nirvn Which screecast mobile app do you recommend ? There are plenty with adds and abo

**Comment by nirvn** [comment]
@jpdupuy , this is a good recorder (https://github.com/you-apps/RecordYou/releases/tag/v8.0). Are you sure your Samsung phone doesn't have a screen recorder? Most of them come with one these days (accessible within the top drawer).

**Comment by jpdupuy** [comment]
@nirvn yes it can capture screenshots, but don't you want a video ?

**Comment by nirvn** [comment]
@jpdupuy , yes, and that's what I meant, most Samsung devices come with a screen video recorder.

**Comment by nirvn** [comment]
@jpdupuy , I'm sorry, it seems the video isn't downloadable anymore, could you re-upload please?

**Comment by jpdupuy** [comment]
@nirvn https://www.swisstransfer.com/d/93d27de0-a76b-485d-ac37-dc1ea21fbe1f

**Comment by nirvn** [comment]
@jpdupuy , since QField 3.6, we have a 3-dot menu button attached to each cloud project with the same actions you'd find on a long press, did that help you?

**Comment by jpdupuy** [comment]
Yes, thank you ... I shall close this issue
